BondageClub/Assets/Female3DCG/Female3DCG.js

```diff
@@ -1466,6 +1466,12 @@ var AssetFemale3DCG = [
 					{ Name: "Shine", AllowColorize: false}
 				]
 			},
+			{ Name: "DildocornHorn", Value: -1, DefaultColor: "#Default", BuyGroup: "UnicornHorn", BodyCosplay: true, Attribute: ["PenetrateItem"], Top: -30,
```

Comment:
`DefaultColor` should be `Default` here - you only need the `#` if you're using hex code colours. The drawing code _should_ handle `#Default` because of guards we put in against invalid colours, but it will likely have to jump through more hoops to do so than if it was just `Default`.
```suggestion
			{ Name: "DildocornHorn", Value: -1, DefaultColor: "Default", BuyGroup: "UnicornHorn", BodyCosplay: true, Attribute: ["PenetrateItem"], Top: -30,
```